test/specs/modules/Dropdown/Dropdown.spec.js

```diff
@@ -54,4 +113,217 @@ describe('Dropdown', () => {
+    options.at(0).trigger('click');
+    expect(wrapper.emitted('input')[0][0]).to.equal(1);
+    options.at(1).trigger('click');
+    expect(wrapper.emitted('input')[1][0]).to.equal(2);
+  });
+
+  it('should choose few options', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        multiple: true,
+        options: [{ text: 'foo', value: 1 }, { text: 'bar', value: 2 }],
+      },
+    });
+
+    wrapper.vm.$on('input', (value) => {
+      wrapper.setProps({ value });
+    });
+
+    const options = wrapper.findAll(DropdownItem);
+
+    options.at(0).trigger('click');
+    expect(wrapper.emitted('input')[0][0]).to.deep.equal([1]);
+    options.at(0).trigger('click');
+    expect(wrapper.emitted('input')[1][0]).to.deep.equal([1, 2]);
+  });
+
+  it('should deselect option from selected', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        multiple: true,
+        options: [{ text: 'foo', value: 1 }, { text: 'bar', value: 2 }],
+      },
+    });
+
+    wrapper.vm.$on('input', (value) => {
+      wrapper.setProps({ value });
+    });
+
+    const options = wrapper.findAll(DropdownItem);
+
+    options.at(0).trigger('click');
+    options.at(0).trigger('click');
+    expect(wrapper.emitted('input')[1][0]).to.deep.equal([1, 2]);
+
+    const selectedOptions = wrapper.findAll(Label);
+
+    selectedOptions.at(0).find('i.icon.delete').trigger('click');
+    expect(wrapper.emitted('input')[2][0]).to.deep.equal([2]);
+  });
+
+  it('should not select more than max-selections', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        multiple: true,
+        maxSelections: 1,
+        options: [{ text: 'foo', value: 1 }, { text: 'bar', value: 2 }],
+      },
+    });
+
+    wrapper.vm.$on('input', (value) => {
+      wrapper.setProps({ value });
+    });
+
+    const options = wrapper.findAll(DropdownItem);
+
+    options.at(0).trigger('click');
+    expect(wrapper.emitted('input')[0][0]).to.deep.equal([1]);
+    options.at(0).trigger('click');
+    expect(wrapper.emitted('input').length).to.equal(1);
+  });
+
+  it('should have icons, flags and images in selected text', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        options: [
+          {
+            text: 'foo',
+            value: 1,
+            flag: 'cn',
+            icon: 'question',
+            image: {
+              src: '/test',
+            },
+          },
+        ],
+      },
+    });
+    wrapper.vm.$on('input', (value) => {
+      wrapper.setProps({ value });
+    });
+
+    wrapper.find(DropdownItem).trigger('click');
+
+    const text = wrapper.find('div.text');
+
+    const icon = text.find(Icon);
+    expect(icon.is(Icon)).to.equal(true);
+    const flag = text.find(Flag);
+    expect(flag.is(Flag)).to.equal(true);
+    const image = text.find(Image);
+    expect(image.is(Image)).to.equal(true);
```

Comment:
For all the expectations like this, you can replace `image.is(Image)` with `image.exists()`